src/odemis/driver/xt_client.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Created on 16 Aug 2019
+
+@author: Thera Pals
+
+Copyright © 2019 Thera Pals, Delmic
+
+This file is part of Delmic Acquisition Software.
+
+Delmic Acquisition Software is free software: you can redistribute it and/or modify it under the terms of the GNU
+General Public License as published by the Free Software Foundation, either version 2 of the License, or (at your
+option) any later version.
+
+Delmic Acquisition Software is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+more details.
+
+You should have received a copy of the GNU General Public License along with Delmic Acquisition Software. If not, see
+http://www.gnu.org/licenses/.
+"""
+from __future__ import division, print_function
+
+import logging
+
+import msgpack_numpy as mn
+import zerorpc
+
+mn.patch()
+
+# Necessary, otherwise nothing at all will be shown
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)-15s: %(message)s")
+logging.getLogger().setLevel(logging.INFO)
```

Comment:
Please don't touch the logging behaviour in a driver/module. Just change it in the "client" (so here, the test case file).